### ISSUE TITLE
Staging Review finding: Incorrect heading levels on review page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105180]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Form 21-4138
**Feature:** Statement In Support of Claim

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/statement-to-support-claim-form-21-4138/review-and-submit](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit), the accordions triggers for reviewing what you've entered are H3s. With "Review and submit" now an H1, we're skipping the H2 heading level.

On the same page, the "Statement of truth" heading is also an H3.

## Recommended action

The accordion triggers should be H2s, as should be the "Statement of truth" heading.

Likewise, the headings inside the accordions will need to be updated to be properly nested when the trigger headings are updated.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
[Issue to track the above](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2023#issue-2848801547)

**Comment by tbaker1026** [comment]
Closed in this[ issue](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2023#issue-2848801547)
